site/profile/templates/slurm/slurm.conf.epp

```diff
@@ -12,7 +12,7 @@ SelectType=select/cons_tres
 SelectTypeParameters=CR_Core_Memory
 
 # NODE CONFIGURATIONS
-GresTypes=gpu
+GresTypes=gpu,shard
```

Comment:
Should it be optional and only set if an instance has its GPU sharded? Or we should not bother and it does not matter if no node has shard defined?

Reply:
It doesn't matter. I haven't found a reason to remove it when shard is not active.